docs/analysis_pipeline.rst

```diff
@@ -187,9 +187,10 @@ Generate figures
 ----------------
 
 Generate figures based on the output csv files using ``sg_generate_figures.py`` script. Run this script in ``/results``
-folder (folder containing csv files) or specify this folder using ``-path-results`` flag. Figures will be created in the
-folder `results/`:
+folder (folder containing csv files) or specify this folder using ``-path-results`` flag. The flag ``-config`` is
+mandatory and an yml file containing the subjects/sites that want to be excluded have to be provided. Figures will 
```

Comment:
```suggestion
mandatory and a yml file containing the subjects/sites that want to be excluded have to be provided. Figures will 
```

Reply:
Fixed in https://github.com/spine-generic/spine-generic/pull/168/commits/df7ba948a6d791624ebc3fd9abeec4904cc1403a